[2024-04-11T23:58:28.053+0000] {processor.py:161} INFO - Started process (PID=228) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:58:28.056+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:58:28.071+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:28.068+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:58:28.162+0000] {processor.py:840} INFO - DAG(s) 'tutorial_dag' retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:58:29.041+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:29.036+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:tutorial_dag
[2024-04-11T23:58:29.049+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:29.049+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:tutorial_dag
[2024-04-11T23:58:29.053+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:29.053+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:tutorial_dag
[2024-04-11T23:58:29.054+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:29.054+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-11T23:58:29.070+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:29.070+0000] {dag.py:3118} INFO - Creating ORM DAG for tutorial_dag
[2024-04-11T23:58:29.090+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:29.090+0000] {dag.py:3954} INFO - Setting next_dagrun for tutorial_dag to 2024-04-11 00:30:00+00:00, run_after=2024-04-11 01:30:00+00:00
[2024-04-11T23:58:29.109+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 1.093 seconds
[2024-04-11T23:58:34.263+0000] {processor.py:161} INFO - Started process (PID=239) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:58:34.265+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:58:34.269+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:34.268+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:58:34.294+0000] {processor.py:840} INFO - DAG(s) 'tutorial_dag' retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:58:34.315+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:34.315+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-11T23:58:34.354+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:58:34.354+0000] {dag.py:3954} INFO - Setting next_dagrun for tutorial_dag to 2024-04-11 22:30:00+00:00, run_after=2024-04-11 23:30:00+00:00
[2024-04-11T23:58:34.377+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 0.125 seconds
[2024-04-11T23:59:04.749+0000] {processor.py:161} INFO - Started process (PID=250) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:04.751+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:59:04.754+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:04.754+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:04.772+0000] {processor.py:840} INFO - DAG(s) 'tutorial_dag' retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:04.920+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:04.920+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-11T23:59:04.949+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:04.949+0000] {dag.py:3954} INFO - Setting next_dagrun for tutorial_dag to 2024-04-11 22:30:00+00:00, run_after=2024-04-11 23:30:00+00:00
[2024-04-11T23:59:04.975+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 0.238 seconds
[2024-04-11T23:59:06.897+0000] {processor.py:161} INFO - Started process (PID=255) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:06.905+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:59:06.911+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:06.910+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:06.940+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:06.932+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/tutorial_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial_dag.py", line 12, in <module>
    buscar_pokemons = PythonOperator(
                      ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-04-11T23:59:06.942+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:07.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 0.142 seconds
[2024-04-11T23:59:37.513+0000] {processor.py:161} INFO - Started process (PID=266) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:37.515+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:59:37.519+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:37.518+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:37.537+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:37.534+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/tutorial_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial_dag.py", line 12, in <module>
    buscar_pokemons = PythonOperator(
                      ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 467, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-04-11T23:59:37.538+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:37.576+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 0.077 seconds
[2024-04-11T23:59:45.834+0000] {processor.py:161} INFO - Started process (PID=271) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:45.837+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:59:45.841+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:45.840+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:45.865+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:45.861+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/tutorial_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial_dag.py", line 12, in <module>
    consumir_api_pokemons = PythonOperator(
                            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 484, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 222, in __init__
    raise AirflowException("`python_callable` param must be callable")
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2024-04-11T23:59:45.866+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:45.888+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 0.067 seconds
[2024-04-11T23:59:52.992+0000] {processor.py:161} INFO - Started process (PID=276) to work on /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:52.994+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/tutorial_dag.py for tasks to queue
[2024-04-11T23:59:52.998+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:52.997+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:53.020+0000] {logging_mixin.py:188} INFO - [2024-04-11T23:59:53.011+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/tutorial_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/tutorial_dag.py", line 14, in <module>
    python_callable = consumir_api_pokemons
                      ^^^^^^^^^^^^^^^^^^^^^
NameError: name 'consumir_api_pokemons' is not defined
[2024-04-11T23:59:53.027+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/tutorial_dag.py
[2024-04-11T23:59:53.062+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/tutorial_dag.py took 0.084 seconds
